types/network/securitylevel.pp

```diff
@@ -0,0 +1,2 @@
+#
+type Collectd::Network::SecurityLevel = Enum['Encrypt', 'Sign', 'None']
```

Comment:
should we namespace this or use a shorter name, like: Collectd::SecurityLevel? That would keep the lines in the Puppet files shorter.

Reply:
I was trying to keep it consistent...I'm not sure that you can avoid many long lines when using types though.